content/en/docs/v3.5/install.md

```diff
@@ -9,86 +9,160 @@ minGoVers: 1.20
 
 Before installing etcd, see the following pages:
 
-- [Supported platforms][]
-- [Hardware recommendations][]
+- [Supported platforms](/docs/v3.5/op-guide/supported-platform)
+- [Hardware recommendations](/docs/v3.5/op-guide/hardware)
 
 ## Install pre-built binaries
 
-The easiest way to install etcd is from pre-built binaries:
+Install `etcd` from pre-built binaries using cURL.
 
- 1. Download the compressed archive file for your platform from [Releases][],
-    choosing release [{{< param git_version_tag >}}][tagged-release] or later.
- 2. Unpack the archive file. This results in a directory containing the binaries.
- 3. Add the executable binaries to your path. For example, rename and/or move
-    the binaries to a directory in your path (like `/usr/local/bin`), or add the
-    directory created by the previous step to your path.
- 4. From a shell, test that `etcd` is in your path:
+   ```bash
+   # Download and extract the latest release using cURL
```

Comment:
Here, good idea to add in the comment v3.5.21 as an example as it will not be the latest release always. So something like, "# Download and extract the latest release using cURL. For example, download v3.5.21 as below."